pkg/combustion/osfiles_test.go

```diff
@@ -0,0 +1,73 @@
+package combustion
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/suse-edge/edge-image-builder/pkg/image"
+)
+
+func setupOsFilesConfigDir(t *testing.T, empty bool) (ctx *image.Context, teardown func()) {
+	ctx, teardownCtx := setupContext(t)
```

Comment:
We have already defined these variables as returned ones so we can just do:
```suggestion
	ctx, teardown = setupContext(t)
```

Reply:
Ah right. I initially started out without the named return values but hit some sort of hiccup and reverted back to that without fixing this.